src/redux/products/products.sagas.js

```diff
@@ -37,8 +40,8 @@ export function* handleFilteredProductsLoad({ payload: { forSearchBar } }) {
       yield put(setProductsLoading(true));
     }
 
-    const state = yield select((state) => state.Products);
-    const currency = yield select((state) => state.Currency.currency);
+    const state = yield select(StateProductsSelector);
```

Comment:
Make sense to rename this variable to `selectProductsState`

Reply:
fixed